src/lib/viewers/BaseViewer.js

```diff
@@ -33,6 +33,7 @@ import {
 } from '../constants';
 import { getIconFromExtension, getIconFromName } from '../icons/icons';
 import { VIEWER_EVENT, ERROR_CODE, LOAD_METRIC, DOWNLOAD_REACHABILITY_METRICS } from '../events';
+import { AnnotationMode } from '../AnnotationControls';
```

Comment:
Should we expose an `exit` or `reset` method to avoid exposing the internals of the controls here?